storm-client/src/jvm/org/apache/storm/task/IOutputCollector.java

```diff
@@ -30,4 +31,5 @@
     void ack(Tuple input);
     void fail(Tuple input);
     void resetTimeout(Tuple input);
+    void flush();
```

Comment:
It is kind of external facing API as it is given to bolt implementors in prepare [API](https://github.com/apache/storm/blob/master/storm-client/src/jvm/org/apache/storm/task/IBolt.java#L55). `IOutputCollector` can be wrapped by users to do any custom processing before they are emitted to target tasks.